src/components/AddPost/AddPost.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+
+import { Avatar, Fab, Tooltip, Typography, styled } from "@mui/material";
+import { Box, Stack } from "@mui/system";
+
+import Modal from "@material-ui/core/Modal";
+import TextField from "@material-ui/core/TextField";
+import ButtonGroup from "@material-ui/core/ButtonGroup";
+import Button from "@material-ui/core/Button";
+
+import AddIcon from "@material-ui/icons/Add";
+import EmojiEmotionsIcon from "@material-ui/icons/EmojiEmotions";
+import ImageIcon from "@material-ui/icons/Image";
+import DateRangeIcon from "@material-ui/icons/DateRange";
+import VideocamIcon from "@material-ui/icons/Videocam";
+import PersonAddIcon from "@material-ui/icons/PersonAdd";
```

Comment:
MUI material ka v5 kiya hua aapne. Koshish karen ke v5 hi ho. But if there isn't enough time then this is fine.